js/controller.js

```diff
@@ -61,9 +61,23 @@
                     console.log(error);
                 });
 
+                setTimeout(function() { refreshFitbitData(); }, 1000);
```

Comment:
Does it need to refresh this often? Maybe you could have it refresh with everything in `refreshMirrorData()` and have a voice command to manually refresh the data.